src/modules/pos/reward/__fixtures__/mockRewardsInflation.js

```diff
@@ -0,0 +1,7 @@
+export const mockRewardsInflation = {
```

Comment:
reward inflation is part of `src/modules/reward`

Reply:
In the other PR i made the fix to this branch. I just merged that PR to this branch. Please recheck now